css/index.css

```diff
@@ -111,26 +121,111 @@ p {
     align-items: center;
     justify-content: center;
 }
-
```

Comment:
With these boxes, you already have almost all of the CSS needed in the parent.  The parent cascades down into the child so the only thing that needs to be done in each child is overriding the color of the box.